packages/@react-spectrum/color/docs/ColorField.mdx

```diff
@@ -0,0 +1,150 @@
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/color';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/color/package.json';
+
+```jsx import
+import {ColorField} from '@react-spectrum/color';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Color
+keywords: [color field, input]
+---
+
+# ColorField
+
+<p>{docs.exports.ColorField.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ColorField']}
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<ColorField label="Primary Color" />
+```
+
+## Value
+
+A ColorField's `value` is empty by default, but an initial, uncontrolled, value can be provided using the `defaultValue` prop.
+Alternatively, a controlled value can be provided using the `value` prop. These values must begin with a `#`.
+
+```tsx example
+function Example() {
+  let [value, setValue] = React.useState('#e73623');
+
+  return (
+    <Flex gap="size-150" wrap>
+      <ColorField
+        label="Primary Color (Uncontrolled)"
+        defaultValue="#e73623" />
+
+      <ColorField
+        label="Primary Color (Controlled)"
+        value={value}
+        onChange={setValue} />
+    </Flex>
+  );
+}
+```
+
+Placeholder text, provided using the placeholder prop, describes the expected value or formatting for the ColorField.
+Placeholder text will only appear when the ColorField is empty, and should not be used as a substitute for labeling the
+component with a visible label.
+
+<ColorField placeholder="#e73623" label="Primary Color" />
+
+## Labeling
+
+A visual label should be provided for the ColorField using the `label` prop. If the ColorField is required, the `isRequired` and
+`necessityIndicator` props can be used to show a required state.
+
+```tsx example
+<Flex gap="size-150" wrap>
+  <ColorField label="Primary Color" />
+  <ColorField label="Primary Color" isRequired />
+  <ColorField label="Primary Color" isRequired necessityIndicator="label" />
+  <ColorField label="Primary Color" necessityIndicator="label" />
+</Flex>
+```
+
+### Accessibility
+
+If a visible label isn't specified, an `aria-label` must be provided to the ColorField for
+accessibility. If the field is labeled by a separate element, an `aria-labelledby` prop must be provided using
+the `id` of the labeling element instead.
+
+### Internationalization
+
+In order to internationalize a ColorField, a localized string should be passed to the `label` or `aria-label` prop.
+When the `necessityIndicator` prop is set to `"label"`, a localized string will be provided for `"(required)"` or `"(optional)"` automatically.
+
+## Step values
```

Comment:
This behavior is different than NumberInput step and was documented how it works. Confirming that we want this difference.